CHANGELOG.md

```diff
@@ -32,6 +32,8 @@
 
 ### Fixes
 
+* `[expect]` Improve output format for mismatchedArgs in mock/spy calls.
```

Comment:
This is probably a "feature" not a "fix"

Reply:
✅